db/migrate/20221219205356_archive_needs_help2017_column.rb

```diff
@@ -0,0 +1,18 @@
+class ArchiveNeedsHelp2017Column < ActiveRecord::Migration[7.0]
+  def change
+    add_column :intake_archives, :needs_help_2017, :integer
+    reversible do |direction|
+      direction.up do
+        safety_assured do
+          # Safe for migrations because this is an INSERT into a column we are creating in this transactional migration.
+          execute <<~SQL
```

Comment:
One downside of this approach is there's no clarity about _which_ product year should expect _which_ columns to exist.